src/email/logs.contoller.js

```diff
@@ -40,7 +40,9 @@
 
         var searchUpdateTimeout;
         function updateSearch(searchData){
-            LoadingModal.open();
+            setTimeout(function() {
```

Comment:
I think both loading modal and the search should use the same timeout. Any reason this is split?